tests/specs/api.e2e.js

```diff
@@ -1,17 +1,7 @@
-const { getTime } = require('../functional/uncat-internal-help.cjs')
-const { getErrorsFromResponse } = require('../functional/err/assertion-helpers')
+import { config } from '../util/defaults'
```

Comment:
Instead of necessitating this in every file, wouldn't it be more graceful to just change the default behavior to use this "default" config here --> https://github.com/newrelic/newrelic-browser-agent/pull/536/files#diff-51da539cea54b4b3bf642582b196cf964b82b2b219d472c1dc68d574da65013eR246-R254

Then the only time you pass a differing config in a test is to override the default.

Thoughts?

Reply:
Yup could do that